php/commands/db.php

```diff
@@ -52,6 +52,10 @@ public function drop( $_, $assoc_args ) {
 	 *
 	 * [--yes]
 	 * : Answer yes to the confirmation message.
+	 * 
+	 * ## EXAMPLES
+	 *
+	 *     wp db reset --yes
```

Comment:
Updated (and added the same example for `wp db drop --yes`. Do you want me to keep the `$`? Seems like I couldn't find any examples containing it.